apps/web/src/app/rsvp/page.tsx

```diff
@@ -41,15 +41,25 @@ export default async function RsvpPage({
 	}
 
 	const rsvpEnabled = await kv.get("config:registration:allowRSVPs");
```

Comment:
Potential Optimization: I like the flow of checking to see if rsvp is enabled first, but since we already grabbed it, might be nice to check this ahead of time to save us a second Redis call, but most importantly, saving us this aggregate db call. Probably not a huge issue, but could be nice considering this page will be called pretty often at one time.  This could be something like short leveraging short circuit logic and doing something like `if ((isRSVPEnabled && await isUnderMaxRSVPLimit()) || user.isRSVPed === true)` could be pretty useful here.

Reply:
331597a fixes this